github/orgs/conftest.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# PyTest support for the various GitHub organization checks
+
+# TODO: convert to logger output
+# TODO: add sleep_* for 'core' functionality
+
+from functools import lru_cache
+from github.branches.validate_compliance import Criteria
+from github.orgs.retrieve_github_data import OrgInfo
+import os
+import pathlib
+
+from typing import List, Set
+
+import subprocess  # nosec
+
+from sgqlc.endpoint.http import HTTPEndpoint
+
+from conftest import METADATA_KEYS  # noqa: I900
+
+
+from . import retrieve_github_data
+import conftest
+
+METADATA_KEYS.update(OrgInfo.metadata_to_log())
+METADATA_KEYS.update(Criteria.metadata_to_log())
+
+
+def orgs_to_check() -> Set[str]:
```

Comment:
Isn't this a duplicate of `_orgs_to_check()` in `github/orgs/retrieve_github_data.py`?

Reply:
Yeah - good catch -- removed this one in next commit, as it was unused.